pytket/pytket/wasm/wasm.py

```diff
@@ -68,6 +68,7 @@ def __init__(self, filepath: str, check_file: bool = True, int_size: int = 32):
 
         function_signatures: list = []
         function_names: list = []
+        _func_lookup = {}
```

Comment:
Can we please add a test that shows the issue this is fixing (i.e. fails before the fix)?

Reply:
I have added one additional file that is now read in